pkg/logger/logger.go

```diff
@@ -132,13 +132,26 @@ func (l *Logger) Output(calldepth int, message string) {
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
-	l.stdLogTemplate.Execute(l.writer, stdLogMessageData{
+	err := l.stdLogTemplate.Execute(l.writer, stdLogMessageData{
 		Timestamp: FormatTimestamp(now),
 		File:      file,
 		Message:   message,
 	})
+	// Fallback for template errors
+	if err != nil {
+		_, ferr := fmt.Fprintf(l.writer, "[%s] [%s] %s",
+			FormatTimestamp(now),
+			file,
+			message)
+		if ferr != nil {
+			panic(ferr)
+		}
+	}
```

Comment:
Hmm, I'm not sure about this, would you not expect a problem with the logger to also cause an issue here? I wonder if the template engine wraps errors so we could tell the difference between some different ones and handle them separately

Reply:
The templating function looks like it can error in 2 ways after digging into it. Either a template issue (which this fallback to Fprintf to the `io.Writer` directly attempts to handle).

Or an issue with the `io.Writer` write itself.

My thinking was if the `io.Writer` is botched, it would fail in both the original and in this `Fprintf` attempt on that handle, so I'd try to write again and in the end panic.

I think if we try to handle the template err and split on the conditions, in the failed template condition I'd still need to check for an io.Writer error at that layer (and then panic still).  I think this waterfall approach covers all the bases we'd have to cover anyway without more branching complexity.